test/type/_normalize.scss

```diff
@@ -6,25 +6,26 @@
 @use '../../../sass/tokens/api';
 @use '../../../sass/type/normalize';
 
-// Test Font Normalization
-// =======================
+$_global: config.$fonts;
+config.$fonts: data.$fonts;
+
 @include test-module('Type: Normalize') {
   // Normalize Font [function]
   // -------------------------
   @include describe('Normalize Font [function]') {
     $test: normalize.normalize-font(map.get(data.$fonts, 'TestFont'));
 
-    /*    @include it('Sets an explicit name, if missing') {
+    @include it('Sets an explicit name, if missing') {
       @include assert-equal(map.get($test, 'name'), 'TestFont');
-    } */
+    }
 
-    /*     @include it('Generates a properly-formatted stack') {
+    @include it('Generates a properly-formatted stack') {
       @include assert-equal(
         map.get($test, 'stack'),
-        ('TestFont', Georgia, serif)
+        (TestFont, Georgia, serif)
```

Comment:
The quoting and unquoting of names in a font stack is important. Output should only have quotes when the font name in question has spaces.